docs/cli.md

```diff
@@ -12,13 +12,15 @@ NAME:
    config-server-encrypt-value - Encrypt a string using a Spring Cloud Services configuration server
 
 USAGE:
-      cf config-server-encrypt-value CONFIG_SERVER_INSTANCE_NAME VALUE_TO_ENCRYPT
+      cf config-server-encrypt-value CONFIG_SERVER_INSTANCE_NAME [VALUE_TO_ENCRYPT]
 
-Note: if VALUE_TO_ENCRYPT contains characters that are special to the shell, it may be necessary
-to use escape characters or quotes. Refer to the shell's man page for details.
+      NOTE: Either VALUE_TO_ENCRYPT or --file-to-encrypt flag is required, but not both.
 
 ALIAS:
    csev
+
+OPTIONS:
+   --f/--file-to-encrypt      A text file whose contents are to be encrypted. Cannot be used with VALUE_TO_ENCRYPT parameter.
```

Comment:
This text is silent about the encoding that is expected for the text file. Ideally, I think this should be defined so that users know which encoding to use. Is it `UTF-8`, for example? The encoding is the one that is specified, or more likely taken as default, when the text is sent to the config server for encryption.

This [article](https://www.joelonsoftware.com/2003/10/08/the-absolute-minimum-every-software-developer-absolutely-positively-must-know-about-unicode-and-character-sets-no-excuses/) is useful for anyone unfamiliar with encodings.

Reply:
It's my understanding (could be wrong) that Go principally deals with UTF-8 (with some limited support for UTF-16) and that working with any other encoding requires some additional packages.

Therefore, unless I'm terribly mistaken (a possibility), the encoding should be UTF-8. For documentation's sake, it's a reasonable qualification, and I've made such changes. If it turns out that other encodings are also suitable, then we can alter the documentation accordingly.

Reply:
Sounds good. Thanks.